cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/sql/SQLBackend.java

```diff
@@ -34,37 +34,38 @@
 
     /**
      * Creates a table, given its name, if not exists in the given database.
-     * @param destination
+     * @param dataBase
      * @param tableName
      * @param fieldNames
      * @throws com.telefonica.iot.cygnus.errors.CygnusRuntimeError
      * @throws com.telefonica.iot.cygnus.errors.CygnusPersistenceError
      */
-    void createTable(String destination, String tableName, String fieldNames)
+    void createTable(String dataBase, String schema, String tableName, String fieldNames)
             throws CygnusRuntimeError, CygnusPersistenceError;
 
     /**
      * Insert already processed context data into the given table within the given database.
-     * @param destination
+     * @param dataBase
+     * @param schema
      * @param tableName
      * @param fieldNames
      * @param fieldValues
      * @throws com.telefonica.iot.cygnus.errors.CygnusBadContextData
      * @throws com.telefonica.iot.cygnus.errors.CygnusRuntimeError
      * @throws com.telefonica.iot.cygnus.errors.CygnusPersistenceError
      */
-    void insertContextData(String destination, String tableName, String fieldNames, String fieldValues)
+    void insertContextData(String dataBase, String schema, String tableName, String fieldNames, String fieldValues)
             throws CygnusBadContextData, CygnusRuntimeError, CygnusPersistenceError;
 
     /**
      * Caps records from the given table within the given database according to the given maximum number.
-     * @param destination
+     * @param dataBase
      * @param tableName
      * @param maxRecords
      * @throws com.telefonica.iot.cygnus.errors.CygnusRuntimeError
      * @throws com.telefonica.iot.cygnus.errors.CygnusPersistenceError
      */
-    void capRecords(String destination, String tableName, long maxRecords) throws CygnusRuntimeError, CygnusPersistenceError;
+    void capRecords(String dataBase, String tableName, long maxRecords) throws CygnusRuntimeError, CygnusPersistenceError;
```

Comment:
schema?

Reply:
This method is only implemented by MySQL sink, That's why I didn't add the schema param. Do you thing It should be included?